modules/ding_event/ding_event.admin.inc

```diff
@@ -6,27 +6,44 @@
 
 /**
  * Get a form to select currency (Euro | Kr)
- * @param $form
- * @param $form_state
+ *
+ * @param array $form
+ *   An associative array containing the structure of the form.
+ * @param array $form_state
+ *   A keyed array containing the current state of the form.
  * @return mixed
  */
-function ding_event_admin_currency_setting_form($form, &$form_state) {
+function ding_event_admin_setting_form(array $form, array &$form_state) {
   $currency = variable_get('ding_event_currency_type', 'Kr');
 
-  $form['ding_event_currency'] = array(
+  $form['ding_event'] = array(
     '#type' => 'fieldset',
     '#title' => t('Event currency'),
     '#tree' => FALSE,
-    '#description' => t("Select which currency to use on events"),
+    '#description' => t("Administer events settings"),
   );
 
-  $form['#submit'][] = 'ding_event_admin_currency_submit';
+  $form['#submit'][] = 'ding_event_admin_submit';
+  $form['#validate'][] = 'ding_event_admin_validate';
 
-  $form['ding_event_currency']['ding_event_currency_type'] = array(
+  $form['ding_event']['ding_event_currency_type'] = array(
     '#type' => 'radios',
     '#title' => t('Currency'),
     '#options' => drupal_map_assoc(array('kr', '€')),
     '#default_value' => $currency,
+    '#description' => t("Select which currency to use on events"),
+  );
+
+  $view = views_get_view('ding_event');
+  $view_items_per_page = $view->display['ding_event_list_frontpage']->display_options['pager']['options']['items_per_page'];
+  $default_items_per_page = variable_get('ding_event_frontend_items_per_page', $view_items_per_page);
```

Comment:
`frontend` is not the same as `frontpage`. The current name indicates that it spans the entire frontend but it only affects the front page.